src/one/nio/serial/Before.java

```diff
@@ -0,0 +1,15 @@
+package one.nio.serial;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Constructor calls are skipped on JSON deserialization unless annotated with this annotation.
+ */
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ElementType.CONSTRUCTOR})
+public @interface Before {
```

Comment:
Is it not enough @Default annotation on fields?

Reply:
Hm, I'll get back to you on this one.